Tests/UITests/TooltipViewSpec.swift

```diff
@@ -0,0 +1,59 @@
+import XCTest
+import Quick
+import Nimble
+import class Shock.MockServer
+
+#if canImport(RSDKUtilsMain)
+import RSDKUtilsMain // SPM version
+#else
+import RSDKUtils
+#endif
+
+class ToolTipViewSpec: QuickSpec {
+    override func spec() {
+
+        var mockServer: MockServer!
+        var app: XCUIApplication!
+        var iamView: XCUIElement {
+            app.otherElements.element(matching: NSPredicate(format: "identifier BEGINSWITH[cd] 'IAMView'"))
+        }
+
+        func launchAppIfNecessary(context: String) {
+            mockServer.setup(route: MockServerHelper.pingRouteMock(jsonStub: context))
+            mockServer.start()
+
+            guard app == nil || !app.launchArguments.joined(separator: " ").contains(context) else {
+                return
+            }
+            self.continueAfterFailure = false
+            app = XCUIApplication()
+            app.launchArguments.append("--uitesting")
+            app.launchArguments.append("-context \(context)")
+            app.launch()
```

Comment:
Because we update the Sample app to support SwiftUI, please add:
```suggestion
            app.launch()
            app.buttons["UIKit"].tap()
```
So the test can be continue.